src/main/kotlin/com/expediagroup/sdk/core/constant/Constant.kt

```diff
@@ -22,6 +22,20 @@ internal object Constant {
 
     const val BEARER = "Bearer"
 
+    const val KEY = "key"
```

Comment:
1. Can we have these constants in a new class?
2. We may consider more specific names to `DEFAULT` and `FILE_SYSTEM `.